docs/rules/order.md

```diff
@@ -164,6 +163,68 @@ import index from './';
 import sibling from './foo';
 ```
 
+### `alphabetize: object`:
+
+Sort the order within each group in alphabetical manner:
+
+- `order`: use `asc` to sort in ascending order, and `desc` to sort in descending order (default: `ignore`).
+- `ignoreCase` [boolean]: when `true`, the rule ignores case-sensitivity of the import name (default: `false`).
+
+Example setting:
+```js
+alphabetize: {
+  order: 'asc', /* sort in ascending order. Options: ['ignore', 'asc', 'desc'] */
+  ignoreCase: false, /* case-sensitive. This property does not have any effect if 'order' is set to 'ignore' */
+}
+```
+
+This will fail the rule check:
+
+```js
+import foo from 'foo';
+import bar from 'bar';
+import Baz from 'Baz';
```

Comment:
These examples make it unclear whether the sorting happens by module names or by bindings names. Making them different would help (this applies to all new examples in this file).

Reply:
And here I though the code is all good and I just need to rebase it.
Totally agree with you about this one